packages/third-parties/agenda/src/AgendaModule.ts

```
@@ -1,8 +1,9 @@
~import {Constant, DIContext, Inject, InjectorService, Module, OnDestroy, Provider, runInContext} from "@tsed/di";
~import {Logger} from "@tsed/logger";
 import {AfterListen} from "@tsed/platform-http";
 import {Job, Processor} from "agenda";
 import {v4 as uuid} from "uuid";
+import type { Collection, Db, Document, Filter } from "agenda/node_modules/mongodb";
```

ESLint

[error] Replace `·Collection,·Db,·Document,·Filter·` with `Collection,·Db,·Document,·Filter`

(prettier/prettier)